### ISSUE TITLE
data.tailscale_device is not working well

### ISSUE THREAD

**Issue by dmitriy-lukyanchikov** [davidsbond/terraform-provider-tailscale#24]
**Describe the bug**
data resource "tailscale_device" is not working for some reason

**To Reproduce**
create code with terraform like this
```
data "tailscale_device" "sample_device" {
  name = "user1-device.example.com"
}
```
and getting error like this
```
╷
│ Error: Failed to fetch devices
│ 
│   with data.tailscale_device.sample_device,
│   on main.tf line 1, in data "tailscale_device" "sample_device":
│    1: data "tailscale_device" "sample_device" {
│ 
│ parsing time """" as ""2006-01-02T15:04:05Z07:00"": cannot parse """ as "2006"
╵
```

**Expected behaviour**
expected to have some output and success result

**Desktop (please complete the following information):**
 - OS: [e.g. MacOS]
 - Terraform Version [e.g. 1.0.4]
 - Provider Version [e.g. 0.1.1]

**Comment by davidsbond** [comment]
Hey @dmitriy-lukyanchikov, thanks for the report. I'll try to take a look at this later today

**Comment by davidsbond** [comment]
Hey @dmitriy-lukyanchikov, I've removed the fields returned from the Tailscale API that were causing this problem. In short, there were some fields assumed to be a `time.Time` type that would return blank strings, which cannot be parsed correctly.

These fields were never exposed in the actual provider, so I've just removed everything except the id and name fields for that type. I've just tagged v0.1.2, which should resolve your problem. It should be available in the provider registry shortly.
